app/assets/javascripts/admin/addon/controllers/admin-web-hooks-index.js

```diff
@@ -0,0 +1,36 @@
+import Controller, { inject as controller } from "@ember/controller";
+import I18n from "I18n";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
+import { alias } from "@ember/object/computed";
+
+export default Controller.extend({
+  adminWebHooks: controller(),
+  dialog: service(),
+  contentTypes: alias("adminWebHooks.contentTypes"),
+  defaultEventTypes: alias("adminWebHooks.defaultEventTypes"),
+  deliveryStatuses: alias("adminWebHooks.deliveryStatuses"),
+  eventTypes: alias("adminWebHooks.eventTypes"),
+  model: alias("adminWebHooks.model"),
+
+  @action
+  destroy(webhook) {
+    return this.dialog.yesNoConfirm({
```

Comment:
Could also use a `deleteConfirm` dialog here, it would come with a red "Delete" confirmation button.